### ISSUE TITLE
Why need the function of 'blob_utils.get_loss_gradients'? 

### ISSUE THREAD

**Issue by yw155** [facebookresearch/Detectron#659]
Hi @ir413, I would like to ask you that why there is a function of 'blob_utils.get_loss_gradients'. From my understanding, after defining the loss, we just need to deploy the op of 'AddGradientOperators' as follows:

loss = model.AveragedLoss(xent, "loss")
model.AddGradientOperators([loss])

So why do we need the function of 'blob_utils.get_loss_gradients' in the Detectron?

Thank you.

**Comment by ljluestc** [comment]
import caffe2 as c2
from detectron.utils import blob_utils

# Assuming you have already defined your model and loss
model = c2.model_helper.ModelHelper(name="my_model")
xent = model.net.SoftmaxWithLoss(["pred", "labels"], "xent")
loss = model.AveragedLoss(xent, "loss")

# Get the gradient operators and blobs for the loss
grad_map = blob_utils.get_loss_gradients(model, [loss])

# Now you can access the gradient blobs for further computation or analysis
gradient_blob = grad_map[loss]
